src/config.rs

```diff
@@ -920,6 +923,34 @@ impl Config {
     }
 }
 
+fn parse_shell(raw: &String) -> Option<Vec<CString>> {
```

Comment:
Would it make sense to have people break this out in config? I'm not a fan of having this parse function.

```yml
shell:
  program: /bin/bash
  args:
    - foo
    - "bar baz"
```